coderadar-ui/src/app/view/file-view/file-view.component.html

```diff
@@ -26,35 +26,42 @@ <h4 *ngIf="this.currentFileContent.length > 0">
   </div>
   <div #tooltip="matTooltip" matTooltip="{{tooltipMessage}}"></div>
   <div>
-    <div fxFlex="0 0 25%" fxLayout="row" style="overflow: scroll">
+    <div fxFlex="0 0 25%" fxLayout="row" style="overflow: scroll; height: 100%">
```

Comment:
Please remove all inline styles. Inline should only be used if there is no other way to overwrite style defined in the files.

Reply:
I've removed the inline styles in this file. However, removing all inline styles is not part of this PR, but rather a part of #379.